js/data/locale/ak/dateformats.json

```diff
@@ -51,98 +45,276 @@
                 "s": "dd"
             },
             "m": {
-                "f": "MMMM",
-                "l": "MMMM",
+                "s": "MM",
                 "m": "MMM",
-                "s": "MM"
+                "l": "MMMM",
+                "f": "MMMM"
             },
             "y": {
-                "f": "yyyy",
-                "l": "yyyy",
+                "s": "yy",
                 "m": "yyyy",
-                "s": "yy"
+                "l": "yyyy",
+                "f": "yyyy"
+            },
+            "n": {
+                "s": "N",
+                "m": "NN",
+                "l": "MMM",
+                "f": "MMMM"
+            },
+            "w": {
+                "s": "E",
+                "m": "EE",
+                "l": "EEE",
+                "f": "EEEE"
             }
         },
         "time": {
             "12": {
-                "h": "h",
-                "m": "mm",
-                "s": "ss",
+                "ahmsz": "h:mm:ss a z",
```

Comment:
diff is just changed the order..

Reply:
Order will be addressed in a separate PR once everything is in development and all unit tests pass.